src/libexpr-c/nix_api_expr.h

```diff
@@ -129,8 +129,8 @@ nix_err nix_value_call_multi(
  *
  * This function converts these Values into their final type.
  *
- * @note You don't need this function for basic API usage very often, since all functions that return a `Value` call it
- * for you. This function is mainly needed before calling @ref getters.
+ * @note You don't need this function for basic API usage as long as you use functions that return a `Value`. This
+ * function is mainly needed before calling @ref getters.
```

Comment:
```suggestion
 * @note This function is mainly needed before calling @ref getters, but not for API calls that return a `Value`. 
```

Reply:
it is not precisely clear what "to need a function _for_ an API call" means. "[...] but not ~for~ *before* API calls that [...]" might be more precise.

but i think i still prefer my original version for two reasons. first, subjectively, it feels clearer to me because i know, as long as i am only using functions that return a `Value`, i do not need to worry, neither before nor after calling them. second, it keeps the formulation "basic API usage", which was introduced by someone who probably knows the api way better than me and which leaves room for some nonbasic api usage using `Value` returning functions only and still requiring `nix_value_force`. so i feel that the new version makes a stronger claim than i dare make.

Reply:
I would avoid qualifying something as "basic", as I don't know what that's supposed to mean and what would be "non-basic" uses of the API. Tried a clarification of the to-be-factual statements: https://github.com/NixOS/nix/pull/10848